keyboards/pitta/sq80/kb.h

```diff
@@ -0,0 +1,22 @@
+#ifndef KB_H
+#define KB_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
We no longer refer to these as keymaps, but as layouts. 

```suggestion
#define LAYOUT( \
```